### ISSUE TITLE
Warning appears in console

### ISSUE THREAD

**Issue by Chaitanyabsprip** [infinum/floggy#31]
```
../../../Downloads/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_loggy-2.0.0/lib/flutter_loggy/stream_printer.dart:16:52: Warning: Operand of null-aware
operation '!' has type 'List<LogRecord>' which excludes null.
 - 'List' is from 'dart:core'.                                          
 - 'LogRecord' is from 'package:loggy/loggy.dart' ('../../../Downloads/flutter/.pub-cache/hosted/pub.dartlang.org/loggy-2.0.0+1/lib/loggy.dart').
    logRecord.add(<LogRecord>[record, ...logRecord.value!]);  
```

I don't mind ignoring it, but it kinda is annoying after a point. Considering the whole point of the package is to promote cleaner logs, moreover log stuff to stdout.

**Comment by RatakondalaArun** [comment]
Any updates on this? @lukaknezic

**Comment by lukaknezic** [comment]
@RatakondalaArun Hey, This is fixed in PR #34. New version (2.0.1) will be out this week 🙂
